boards/nxp/lpcxpresso55s69/Kconfig.defconfig

```diff
@@ -30,7 +30,7 @@ DT_CHOSEN_Z_CODE_PARTITION := zephyr,code-partition
 
 config FLASH_LOAD_SIZE
 	default $(dt_chosen_reg_size_hex,$(DT_CHOSEN_Z_CODE_PARTITION))
-	depends on BOARD_LPCXPRESSO55S69_LPC55S69_CPU0 && TRUSTED_EXECUTION_SECURE
+	depends on BOARD_LPCXPRESSO55S69_LPC55S69_CPU0_NS && TRUSTED_EXECUTION_SECURE
```

Comment:
This is not true by looking at https://github.com/zephyrproject-rtos/zephyr/blob/086e144be7edab139c386d81e71bbd178bac259a/boards/nxp/lpcxpresso55s69/lpcxpresso55s69_lpc55s69_cpu0_ns_defconfig the ns defconfig disables `TRUSTED_EXECUTION_SECURE` and sets `TRUSTED_EXECUTION_NONSECURE`

Reply:
done